crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -318,31 +328,25 @@ where
             }
         });
 
-        std::thread::spawn({
+        tokio::spawn({
```

Comment:
👍 I don't think that a whole thread for this was needed. Especially with the new runtime (?!)